File: 07. Perulangan Lanjut/Pola III/Pola-III.c
#include <stdio.h>

int main ()
{
    unsigned short bilangan = 0, angka = 0;

    scanf ("%hu", &bilangan);

    for (unsigned short i = 1; i <= bilangan; i++)
    {
        for (unsigned short j = 1; j <= i; j++)
        {
            printf ("%hu", angka);
            angka++;

            if (angka == 10)
            {
                angka = 0;
            }
        }
        printf ("\n");
    }
    return 0;
}

File: 07. Perulangan Lanjut/Pola III/PolaIII.java
import java.util.Scanner;

public class PolaIII
{
    public static void main (String [] args)
    {
        Scanner scanner = new Scanner (System.in);

        short angka = 0;

        short bilangan = scanner.nextShort ();

        scanner.close ();

        for (short i = 1; i <= bilangan; i++)
        {
            for (short j = 1; j <= i; j++)
            {
                System.out.print (angka);
                angka++;

                if (angka == 10)
                {
                    angka = 0;
                }
            }

            System.out.println ();
        }
    }
}
